### PR TITLE
use setuptools_scm to manage the version (via git tags)

### PR DESCRIPTION
**Pull request by nhoening**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,7 +46,8 @@
         "timely_beliefs.examples",
         "timely_beliefs.visualization",
     ],
-    include_package_data=True,
+    packages=find_packages(),  # will include *.py files and some other types
```
Comment (Flix6x) [inline setup.py:49]: I got an error here, when running `to_pypi`: `SyntaxError: keyword argument repeated`. There is already a `packages` key, so I guess you still need to remove lines 40-48?

**Review by Flix6x — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (nhoening) [reply setup.py:49]: Oh, that is right, good catch. We can also choose not add line 49.

Comment (nhoening) [reply setup.py:49]: This is what `find_packages` finds:

```
['timely_beliefs', 'timely_beliefs.tests', 'timely_beliefs.sensors', 'timely_beliefs.sources', 'timely_beliefs.examples', 'timely_beliefs.visualization', 'timely_beliefs.beliefs', 'timely_beliefs.sensors.func_store']
```

Comment (nhoening) [reply setup.py:49]: So I'd argue we keep using it, it's nice.

**Review by Flix6x — APPROVED** [review]

**Pull request merged**